packages/api/src/external/carequality/document/__tests__/create-outbound-document-retrieval-req.test.ts

```diff
@@ -43,7 +43,7 @@ describe("outboundDocumentRetrievalRequest", () => {
     homeCommunityId = faker.string.uuid();
   });
 
-  it("returns zero req when no doc refs matching GW homeCommunityId", async () => {
+  it("returns 1 req when no doc refs matching GW homeCommunityId", async () => {
```

Comment:
its the expected behavior now that we updated this: https://github.com/metriport/metriport/pull/2036/files#r1605495378

Reply:
Basically 1 was passing because the logic in createOuboundDocumentReferences is:

If there are no documentReferences in the request, filter it out. So since we had 2 document references in the request, we went from 0 requests to 1. 

I then also added a check that their are 2 document references in the 1 req and updated the language